rapid7_insightvm_cloud/plugin.spec.yaml

```
@@ -0,0 +1,1202 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_insightvm_cloud
+title: Rapid7 InsightVM Cloud
+description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the InsightVM cloud API to view assets, scan results, and start scans
```

```suggestion
description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the InsightVM Cloud API to view assets, scan results, and start scans
```
@zyoutz-r7 do you know the branding on InsightVM Cloud API? Is it capitalized?